templates/zerver/integrations/redmine.md

```diff
@@ -0,0 +1,48 @@
+Get information on new or updated Redmine issues right in
+Zulip with our Zulip Redmine plugin! Note: this setup must be
+done by a Redmine Administrator.
+
+{!create-stream.md!}
+
+Then:
+
+1. Clone the [Zulip Redmine plugin repository][1] by running:
+   `git clone https://github.com/zulip/zulip-redmine-plugin`
+   [1]: https://github.com/zulip/zulip-redmine-plugin
+
+2. Follow the [Redmine plugin installation guide][2] to install
+   the `zulip_redmine` plugin directory, which is a subdirectory
+   of the `zulip-redmine-plugin` repository directory. In a nutshell,
+   the steps are:
+   [2]: http://www.redmine.org/projects/redmine/wiki/Plugins
+       1. Copy the `zulip_redmine` directory to the `plugins`
```

Comment:
The numbering on these sub-tasks got a bit messed up and displays as 3, 4, 5. 
(And so the last one at the bottom is 10, which formats weirdly from the extra digit)